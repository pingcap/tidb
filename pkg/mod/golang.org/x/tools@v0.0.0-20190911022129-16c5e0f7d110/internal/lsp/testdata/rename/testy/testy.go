package testy

type tt int //@rename("tt", "testyType")

func a() {
	foo := 42 //@rename("foo", "bar")
}
