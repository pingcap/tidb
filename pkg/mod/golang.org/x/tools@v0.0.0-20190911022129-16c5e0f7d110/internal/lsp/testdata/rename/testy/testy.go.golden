-- bar-rename --
testy.go:
package testy

type tt int //@rename("tt", "testyType")

func a() {
	bar := 42 //@rename("foo", "bar")
}

-- testyType-rename --
testy.go:
package testy

type testyType int //@rename("tt", "testyType")

func a() {
	foo := 42 //@rename("foo", "bar")
}

